x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/endpoint_info.tsx

```diff
@@ -7,26 +7,52 @@
 
 import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import React from 'react';
+import { InferenceAPIConfigResponse } from '@kbn/ml-trained-models-utils';
 import { isEndpointPreconfigured } from '../../../../utils/preconfigured_endpoint_helper';
 import * as i18n from './translations';
 
 export interface EndpointInfoProps {
   inferenceId: string;
+  provider: InferenceAPIConfigResponse;
 }
 
-export const EndpointInfo: React.FC<EndpointInfoProps> = ({ inferenceId }) => (
-  <EuiFlexGroup justifyContent="spaceBetween">
-    <EuiFlexItem grow={false}>
-      <span>
-        <strong>{inferenceId}</strong>
-      </span>
-    </EuiFlexItem>
-    <EuiFlexItem grow={false}>
-      <span>
-        {isEndpointPreconfigured(inferenceId) ? (
-          <EuiBetaBadge label={i18n.PRECONFIGURED_LABEL} size="s" color="hollow" />
-        ) : null}
-      </span>
-    </EuiFlexItem>
-  </EuiFlexGroup>
-);
+export const EndpointInfo: React.FC<EndpointInfoProps> = ({ inferenceId, provider }) => {
+  const taskType = provider.task_type;
+  const isInTechPreview =
+    taskType === 'rerank' &&
+    // @ts-ignore
+    provider.service_settings?.model_id?.startsWith('.');
```

Comment:
IMO We should write a function for this, and ideally fix it to not require a `ts-ignore`

`function isProviderTechPreview(provider: InferenceAPIConfigResponse): boolean`

Then this can be done just like the usage of `isEndpointPreconfigured`